packages/odyssey-react/.storybook/preview.scss

```diff
@@ -1,53 +1,16 @@
-@import '@okta/odyssey';
-
-@font-face {
-  font-family: 'Whyte';
-  font-style: normal;
-  font-weight: normal;
-  src: url('https://ok3static.oktacdn.com/font/ABCWhyte-Book.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Whyte';
-  font-style: italic;
-  font-weight: normal;
-  src: url('https://ok3static.oktacdn.com/font/ABCWhyte-BookItalic.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Whyte';
-  font-style: normal;
-  font-weight: 600;
-  src: url('https://ok3static.oktacdn.com/font/ABCWhyte-Medium.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Whyte';
-  font-style: normal;
-  font-weight: 600;
-  src: url('https://ok3static.oktacdn.com/font/ABCWhyte-Medium.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Whyte';
-  font-style: italic;
-  font-weight: 600;
-  src: url('https://ok3static.oktacdn.com/font/ABCWhyte-MediumItalic.woff') format('woff');
-}
-
-@font-face {
-  font-family: 'Whyte';
-  font-style: italic;
-  font-weight: 600;
-  src: url('https://ok3static.oktacdn.com/font/ABCWhyte-MediumItalic.woff') format('woff');
-}
-
-// TODO: Inconsolata was just selected by our brand team as
-// Okta's font of choice. Work to ensure it gets put in the
-// CDN like Whyte has.
-
-@import url('https://fonts.googleapis.com/css2?family=Inconsolata&display=swap');
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
```

Comment:
hmmm wonder how this got missed before? also minor but looks like you may have copy pasted an older copyright header (2018-present) rather than letting the linter fix auto generate the date (2021-present). Was the linter fix problematic for some reason?

Reply:
Good catch! Updating the date.

The linter ignores hidden directories, so none of our `.storybook` files have headers. Since I was touching this one, I figured I might as well add it manually.

Reply:
Thanks for this info - I can look at a fix for that this week